src/pydicom/pixels/utils.py

```diff
@@ -795,21 +795,44 @@ def get_j2k_parameters(codestream: bytes) -> dict[str, object]:
         JPEG 2000 `codestream`, or an empty dict if unable to parse the data.
         Available parameters are ``{"precision": int, "is_signed": bool}``.
     """
+    offset = 0
+    info: dict[str, Any] = {"jp2": False}
+
+    # Account for the JP2 header (if present)
+    # The first box is always 12 bytes long
+    if codestream.startswith(b"\x00\x00\x00\x0C\x6A\x50\x20\x20"):
```

Comment:
I would have thought that with such a relatively new standard all vendors will create conform DICOM... ah well...